package.json

```diff
@@ -12,7 +12,9 @@
   "devDependencies": {
     "@babel/cli": "^7.7.0",
     "@babel/core": "^7.7.2",
-    "@babel/eslint-plugin-development": "^1.0.1",
+    "@babel/eslint-config-internal": "./eslint/babel-eslint-config-internal",
```

Comment:
Is this the best way to use these internal packages for linting before we publish them? Or is there another way?

Reply:
Hmmm okay, so this doesn't actually work because it only copies in `src`. Will have to keep digging.